pkg/installation/install.go

```diff
@@ -94,5 +106,62 @@ func Remove(p environment.KrewPaths, name string) error {
 	}
 	glog.V(1).Infof("Deleting plugin version %s", version)
 	glog.V(3).Infof("Deleting path %q", filepath.Join(p.Install, name))
+	removeLink(p.Bin, name)
 	return os.RemoveAll(filepath.Join(p.Install, name))
 }
+
+func createOrUpdateLink(binDir string, binary string, plugin string) error {
+	dst := filepath.Join(binDir, pluginNameToBin(plugin))
+
+	if err := removeLink(binDir, plugin); err != nil {
+		return fmt.Errorf("failed to remove old symlink, err: %v", err)
+	}
+	if _, err := os.Stat(binary); os.IsNotExist(err) {
+		return fmt.Errorf("can't create file, destination %q does not exsist", binary)
+	}
+
+	// Create new
+	glog.V(2).Infof("Creating symlink from %q to %q\n", binary, dst)
+	if err := os.Symlink(binary, dst); err != nil {
+		return fmt.Errorf("failed to create a symlink form %q to %q, err: %v", binDir, dst, err)
+	}
+	glog.V(2).Infof("Created symlink in %q\n", dst)
+
+	return nil
+}
+
+func checkSubPath(subPath, path string) error {
```

Comment:
FYI it seems like everywhere we call `pathutil.IsSubPath` we're passing filepath.Abs'd strings.

Do you think if we add filepath.Abs _inside_ `IsSubPath` we can avoid (1) creating little methods like this (currently untested) (2) remove code duplication when calling IsSubPath?